### PR TITLE
make data flow between components better on program selection

### PR DESCRIPTION
**Pull request by smithmd**

### CODE REVIEW

File: src/app/components/application/program-type/certificate-info/certificate-info.component.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import {Component, OnInit} from '@angular/core';
+import {Component, Input, input, OnChanges, OnInit, SimpleChanges} from '@angular/core';
```
Comment (imbmoconno) [inline src/app/components/application/program-type/certificate-info/certificate-info.component.ts:1]: Is the `input` intentional?

File: src/app/components/application/program-type/program-type.component.html [hunk 2]
```diff
@@ -24,7 +24,7 @@ <h2 ngbAccordionHeader>
       </h2>
       <div ngbAccordionCollapse>
         <div ngbAccordionBody>
-          <iee-program-info></iee-program-info>
+          <iee-program-info [appDataTime]="appDataTime" [ageGroup]="appData.ageGroup"></iee-program-info>
```
Comment (imbmoconno) [inline src/app/components/application/program-type/program-type.component.html:27]: Should these be `{{appDataTime}}` instead of just a string?

Comment (imbmoconno) [reply src/app/components/application/program-type/program-type.component.html:27]: Ohh nm... I'm dumb lol

File: src/app/components/application/program-type/certificate-info/certificate-info.component.ts (hunk 1 above)
Comment (smithmd) [reply src/app/components/application/program-type/certificate-info/certificate-info.component.ts]: no. fixed

**Review by imbmoconno — APPROVED** [review]

**Pull request merged**